content/How to customize fonts in React Native - Manuela Viola.md

```diff
@@ -0,0 +1,109 @@
+# How to customize fonts in React Native
+
+![Main image](images/customize_fonts_main.png)
+
+## Introduction
+
+Like every newbie in Rootstrap, my first assignment was to develop Target MVD application, in my case, in React Native. While doing this, one of the first things I tried was to set the family font that the application uses, and I found out it wasn't as easy as I expected.
+
+The biggest issue was that all the tutorials said to excecute "react-native link" command and after executing it, the application would stop building successfully.
+
+## Overview
+
+In this tutorial you will learn how to add and link fonts manually, without using the link command.
+
+First you will see all the configurations that need to be done in order to be able to use customize fonts and then you will learn its usage in a simple application.
+
+## Configuration
+
+### Create Project
+
+First of all, you should create a project in react native. To do that, initiate terminal in the folder where you want to create your project and excecute:
+
+```sh
+react-native init customize_fonts_react_native_tutorial
+```
+
+### Install libraries
+
+After creating the project, there are some libraries that need to be installed:
+
+```sh
+cd customize_fonts_react_native_tutorial
+npm install --save react-native-global-props
+npm install --save babel-plugin-module-resolver
+```
+
+### Download and rename fonts
+
+Since this is a tutorial about customizing fonts, you will need to have some fonts downloaded, in order to add them to your React Native project.
+
+There are many places where you can get this customized fonts. For this tutorial, I used two different fonts, [GoodFeelingSans](https://www.wfonts.com/font/good-feeling-sans) and [Dan'sDisney](https://www.fontspace.com/dans-disney-font-f24536). In both cases I ended up with a .ttf file.
+
+For IOS is important the font's filename, because it should be the same as its PostScript name. To know this, you should open your .ttf file with Font Book, see what its PostScript name is and rename the font to it.
```

Comment:
In iOS (lower case i) is important for the font filename to be the same as its PostScript name. 
To find the PostScript name, open the .ttf file with Font Book...